source/features/show-open-prs-of-forks.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import cache from 'webext-storage-cache';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {isRepoWithAccess} from '../libs/page-detect';
+import {getForkedRepo, getUsername} from '../libs/utils';
+
+interface PullRequestData {
+	count: number;
+	firstUrl: string;
+}
+
+function getUserPullRequestsURL(forkedRepo: string, user: string): string {
+	return `https://github.com/${forkedRepo}/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+author%3A${user}`;
+}
+
+const getOpenPullRequestsData = cache.function(async (forkedRepo: string|undefined): Promise<PullRequestData | false> => {
+	if (!forkedRepo || !isRepoWithAccess()) {
+		return false;
+	}
+
+	// Grab the PR count and the first PR's URL
+	// This allows to link to the PR directly if only one is found
+	const {search} = await api.v4(`
+		search(type: ISSUE, query: "repo:${forkedRepo} is:pr is:open author:${getUsername()}", first: 1) {
+			issueCount
+			nodes {
+				... on PullRequest {
+					url
+				}
+			}
+		}
+	`);
+
+	if (search.issueCount === 0) {
+		return false;
+	}
+
+	return {
+		count: search.issueCount,
+		firstUrl: search.nodes[0].url
+	};
+}, {
+	maxAge: 1 / 2, // Stale after 12 hours
+	staleWhileRevalidate: 2,
+	cacheKey: ([forkedRepo]): string => `${__featureName__}:pr-data:${forkedRepo ?? '-'}`
+});
+
+async function initHeadHint(): Promise<void | false> {
+	const forkedRepo = getForkedRepo();
+	const pullRequestsData = await getOpenPullRequestsData(forkedRepo);
+
+	if (!pullRequestsData) {
+		return;
+	}
+
+	const user = getUsername();
+	const textContainer = select('.fork-flag .text')!;
+
+	if (pullRequestsData.count === 1) {
+		textContainer.append(
+			<> with <a href={pullRequestsData.firstUrl}>one open pull request</a></>
+		);
+	} else {
+		const pullRequestsURL = getUserPullRequestsURL(forkedRepo!, user);
+
+		textContainer.append(
+			<> with <a href={pullRequestsURL}>{pullRequestsData.count} open pull requests</a></>
+		);
+	}
+}
+
+async function initDeleteHint(): Promise<void | false> {
+	const forkedRepo = getForkedRepo();
+	const pullRequestsData = await getOpenPullRequestsData(forkedRepo);
+
+	if (!pullRequestsData) {
+		return;
+	}
+
+	const user = getUsername();
+	const deleteDialogParagraph = select('details-dialog[aria-label*="Delete"] .Box-body p:first-child');
+
+	if (!deleteDialogParagraph) {
+		return;
+	}
+
+	const pullRequestsLink = pullRequestsData.count === 1 ?
+		<a href={pullRequestsData.firstUrl}>your open pull request</a> :
+		<a href={getUserPullRequestsURL(forkedRepo!, user)}>your {pullRequestsData.count} open pull requests</a>;
+
+	deleteDialogParagraph.after(
+		<p className="flash flash-warn">
+			It will also abandon {pullRequestsLink} in <strong>{forkedRepo}</strong> and
+			you&apos;ll no longer be able to edit {pullRequestsData.count === 1 ? 'it' : 'them'}.
```

Comment:
```suggestion
			you’ll no longer be able to edit {pullRequestsData.count === 1 ? 'it' : 'them'}.
```

Reply:
That's where I'm coming from, but it will [upset the linter](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md).

Reply:
’ isn't '

It shouldn't complain